CONTRIBUTORS.md

```diff
@@ -11,9 +12,7 @@
 
 -[@varunpusarla](https://github.com/varunpusarla)
 
-
--[@seakun](https://github.com/seakun)
--[@swatiidixit](https://github.com/swatiidixit)
+-[@seakun](https://github.com/seakun) -[@swatiidixit](https://github.com/swatiidixit)
```

Comment:
Bunching other students up like this is not ideal. 
Any chance you could correct this?

Reply:
I'm sorry,
it looks like the last merge from another commit merged the bunched up code. That's probably my fault.
Perhaps @ChrisPfeuffer can remedy that in his next pull request?